### DISCUSSION TITLE
How do I safely shutdown a lotus-worker?

### DISCUSSION THREAD

**Discussion by travisperson** [filecoin-project/lotus#6784] [Storage Provider] [answered]
Marty the miner needs to apply security updates to their operating system and needs to restart their host. How does smarty stop the lotus-miner from scheduling additional jobs onto the worker so that they can safely reboot their hosts without foobaring their miner?

**Comment by raulk** [comment]
@magik6k is this a question, or is it a statement? Do we have a way to safely shutdown a Lotus worker?

**Comment by dd45e640b42e6da7da96faee3996ef7c** [comment, marked as answer]
you can't. just kill it.  duplicate issue 

#5227 

#antidisestablishatarianism
